solutions/standard/outputs.tf

```diff
@@ -59,3 +59,13 @@ output "certificate_base64" {
   value       = module.redis.certificate_base64
   sensitive   = true
 }
+
+output "service_credential_secrets" {
```

Comment:
This is not consistent with elasticsearch which outputs secrets_manager_secrets from the DA.